shared/common-adapters/clickable-box.desktop.js

```diff
@@ -58,8 +62,10 @@ class ClickableBox extends React.Component<Props & {children: any}, {mouseDown:
       <div
         {...otherProps}
         onMouseDown={this._onMouseDown}
-        onMouseEnter={this._onMouseEnter}
-        onMouseLeave={this._onMouseLeave}
+        // Set onMouseEnter/Leave only if needed, so that any hover
+        // properties of children elements work.
+        onMouseEnter={needMouseEnterLeaveHandlers(this.props) ? this._onMouseEnter : undefined}
```

Comment:
Changed this to effectively not set an underlay unless hoverColor/underlayColor are specified. The defaults are so subtle as to be unnoticable, anyway.